lib/internal/async_hooks.js

```diff
@@ -129,6 +129,7 @@ function emitInitNative(asyncId, type, triggerAsyncId, resource) {
   active_hooks.call_depth += 1;
   // Use a single try/catch for all hooks to avoid setting up one per iteration.
   try {
+    // Using var here instead of let because "for (var ...)" is faster than let
```

Comment:
```suggestion
    // Using var here instead of let because "for (var ...)" is faster than let.
    // Refs: https://github.com/nodejs/node/pull/30380#issuecomment-552948364
```